sympy/printing/str.py

```diff
@@ -645,6 +645,9 @@ def _print_Uniform(self, expr):
     def _print_Union(self, expr):
         return ' U '.join(self._print(set) for set in expr.args)
 
+    def _print_SymmetricDifference(self, expr):
+        return ' ^ '.join(self._print(set) for set in expr.args)
```

Comment:
You should remove this function. `^` is not the right symbol for SymmetricDifference. Without any specialized function the result should be something like
```
In [2]: str(SymmetricDifference(Interval(0, 1), Interval(3, 4), evaluate=False))
Out[2]: 'SymmetricDifference([0, 1], [3, 4])'
```